fre/cmor/cmor_mixer.py

```diff
@@ -4,408 +4,383 @@
+                            target_var=None, json_exp_config=None, json_table_config=None,
+                            prev_path=None):
     '''
-    rewrite the input netcdf file nc_fl containing target_var in a CMIP-compliant manner.
-    accepts six arguments, all required:
-        proj_table_vars: json dictionary object, variable table read from json_table_config.
-        local_var: string, variable name used for finding files locally containing target_var,
-                   this argument is often equal to target_var.
-        netcdf_file: string, representing path to intput netcdf file.
+    Rewrite the input netcdf file containing target_var in a CMIP-compliant manner.
+
+    Parameters:
+        mip_var_cfgs: json dictionary object, variable table read from json_table_config.
+        local_var: string, variable name used for finding files locally containing target_var.
+        netcdf_file: string, representing path to input netcdf file.
         target_var: string, representing the variable name attached to the data object in the netcdf file.
-        json_exp_config: string, representing path to json configuration file holding metadata for appending to output
-                         this argument is most used for making sure the right grid label is getting attached to the right output
-        json_table_config: string, representing path to json configuration file holding variable names for a given table.
-                           proj_table_vars is read from this file, but both are passed anyways.
-    '''
-    fre_logger.info( "input data: " )
-    fre_logger.info(f"    local_var   =  {local_var}" )
-    fre_logger.info(f"    target_var = {target_var}")
+        json_exp_config: string, representing path to configuration file holding metadata for appending to output.
+        json_table_config: string, representing path to configuration file holding variable names for a given table.
 
+    Returns:
+        filename: The name of the file returned by cmor.close.
+    '''
+    fre_logger.info("input data:")
+    fre_logger.info("    local_var   =  %s", local_var)
+    fre_logger.info("    target_var = %s", target_var)
 
     # open the input file
-    fre_logger.info(f" opening {netcdf_file}" )
-    ds = nc.Dataset(netcdf_file,'r+')#'a')
-
+    fre_logger.info("opening %s", netcdf_file)
+    ds = nc.Dataset(netcdf_file, 'r+')
 
     # ocean grids are not implemented yet.
-    fre_logger.info( 'checking input netcdf file for oceangrid condition')
+    fre_logger.info('checking input netcdf file for oceangrid condition')
     uses_ocean_grid = check_dataset_for_ocean_grid(ds)
     if uses_ocean_grid:
-        fre_logger.info('OH BOY you have a file on the native tripolar grid...\n'
-              '... this is gonna be fun!' )
+        fre_logger.info(
+            'OH BOY you have a file on the native tripolar grid...\n'
+            '... this is gonna be fun!'
+        )
```

Comment:
Very funny. We'll tone down the commentary once the tripolar CMORizing is confirmed..